src/components/staking/LiValidator.vue

```diff
@@ -23,15 +29,9 @@
           :class="statusColor"
           class="data-table__row__info__container__status"
         />
-        <router-link
-          :to="{
-            name: 'validator',
-            params: { validator: validator.operator_address }
-          }"
-          class="data-table__row__info__container__name"
-        >
+        <span class="data-table__row__info__container__name">
```

Comment:
this is great, but this validator name shouldn't be blue anymore

Reply:
It is not:
![image](https://user-images.githubusercontent.com/5869273/62059497-6a4a4e00-b1f1-11e9-8b42-dd67d7f4013a.png)